backend/app/transcendence/.eslintrc.js

```diff
@@ -0,0 +1,327 @@
+        "no-use-before-define": "error",
+        // SUGESTED
+        "accessor-pairs":
+        [
+            "error",
+            {
+                "getWithoutSet": true,
+                "setWithoutGet": true,
+                "enforceForClassMembers": false
+            }
+        ],
+        "arrow-body-style":
+        [
+            "error",
+            "always"
+        ],
+        "block-scoped-var": "error",
+        "camelcase":
+        [
+            "error",
+            {
+                "ignoreImports": true
+            }
+        ],
+        "consistent-return": "error",
+        "curly":
+        [
+            "error",
+            "multi"
+        ],
+        "default-case": "error",
+        "default-case-last": "error",
+        "dot-notation": "error",
+        "eqeqeq":
+        [
+            "error",
+            "smart"
+        ],
+        "func-style":
+        [
+            "error",
+            "declaration",
+            {
+                "allowArrowFunctions": true
+            }
+        ],
+        "grouped-accessor-pairs": "error",
+        "init-declarations":
+        [
+            "error",
+            "never",
+            {
+                "ignoreForLoopInit": true
+            }
+        ],
+        "max-classes-per-file": "error",
+        "max-depth":
+        [
+            "error",
+            4
+        ],
+        "max-lines-per-function":
+        [
+            "error",
+            {
+                "max": 25,
+                "skipComments": true,
+                "skipBlankLines": true
+            }
+        ],
+        "max-params":
+        [
+            "error",
+            4
+        ],
+        "max-statements":
+        [
+            "error",
+            4
+        ],
+        // "new-cap": "error",
+        "no-alert": "error",
+        "no-case-declarations": "error",
+        "no-inline-comments": "error",
+        "no-label-var": "error",
+        "no-multi-assign": "error",
+        "no-nested-ternary": "error",
+        "no-new": "error",
+        "no-var": "error",
+        "spaced-comment":
+        [
+            "error",
+            "always"
+        ],
+        "vars-on-top": "error",
+        "yoda":
+        [
+            "error",
+            "never",
+            {
+                "onlyEquality": true
+            }
+        ],
+        // Layout and Formatting
+        "array-element-newline":
+        [
+            "error",
+            "always"
+        ],
+        "array-bracket-newline":
+        [
+            "error",
+            {
+                "multiline" : true
+            }
+        ],
+        "arrow-parens":
+        [
+            "error",
+            "always"
+        ],
+        "block-spacing": "error",
+        "brace-style":
+        [
+            "error",
+            "allman"
+        ],
+        // "comma-dangle":
+        // [
+        //     "error",
+        //     "always"
+        // ],
```

Comment:
Same question, does it serve a purpose to keep this part as commented?

Reply:
We can remove it safely after several usage